packages/ibm-products-styles/src/components/Datagrid/styles/_datagrid.scss

```diff
@@ -158,7 +158,7 @@
 
 /* This section to be removed after support for slug dropped  - start */
 .#{$block-class} th.#{$block-class}__with-slug {
-  /* stylelint-disable-next-line carbon/theme-token-use */
+  /* stylelint-disable-next-line carbon/theme-use */
```

Comment:
Are AI tokens not captured in the `stylelint-plugin-carbon-tokens` package or is this just leftover from maybe an earlier version? 👀